sites/demo/src/frontend/package.json

```diff
@@ -31,9 +33,10 @@
   },
   "homepage": "https://github.com/openfun/richie-site-factory#readme",
   "dependencies": {
-    "richie-education": "2.0.0-beta.15"
+    "richie-education": "2.0.0-beta.17"
```

Comment:
You need to add a line in each CHANGELOG.md file mentioning the upgrade
 (this is not taken care of by the bin/upgrade script yet and must be done manually...)